### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by CodelingoBotTester**

### CODE REVIEW

File: new.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// is_null your comment
+// Some different comment
```
Comment (CodelingoBotTester) [inline new.js]:
```diff
- // Some different comment
+ // Some {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Node.js command failed with output : exec: "parsediff.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Node.js command failed with output : exec: "parsediff.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Node.js command failed with output : exec: "parsediff.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Node.js command failed with output : exec: "parsediffdocker.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Node.js command failed with output : exec: "parsediffdocker.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: error handling failed for:POST https://api.github.com/repos/Daanikus/interactive-rewrite-test/pulls/8/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]: POST https://api.github.com/repos/Daanikus/interactive-rewrite-test/pulls/8/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply new.js]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply new.js]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

File: example.php [hunk 2]
```diff
@@ -1,7 +1,11 @@
+<<<<<<< HEAD
 <?php
 
 $foo = NULL;
 // is_null is a function that does stuff 
 is_null($foo);
 
-?>
\ No newline at end of file
+// Some comment
```
Comment (CodelingoBotTester) [inline example.php]:
```diff
- // Some different comment
+ // Some {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php]: Node.js command failed with output : exec: "parsediff.sh": executable file not found in $PATH

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php]: Success! Committed rewritten file.

**Pull request closed without merging**